doc/lsst.ap.verify/running.rst

```diff
@@ -0,0 +1,80 @@
+.. _ap-verify-running:
+
+#######################################
+Running ap_verify From the Command Line
+#######################################
+
+``ap_verify`` is a Python script designed to be run on both developer machines and verification servers.
+This page describes the minimum options needed to run ``ap_verify``.
+For more details, see the :ref:`ap-verify-cmd` or run ``ap_verify`` with the ``-h`` argument.
+
+.. _ap-verify-dataset-name:
+
+Datasets as Input Arguments
+---------------------------
+
+While ``ap_verify`` is not a :ref:`command-line task<lsst.pipe.base>`, the command-line interface is designed to resemble that of command-line tasks where practical.
+The most important difference is that, since ``ap_verify`` begins with an uningested :ref:`dataset<ap-verify-datasets>`, the input argument is a dataset name rather than a repository.
+
+.. TODO: is the abstraction of having a separate `dataset` name useful, or does it just impose more work on the user (who needs the repo name anyway to install the dataset)? (DM-12853)
+
+Datasets are identified by a name that gets mapped to an :ref:`eups-registered directory <ap-verify-datasets-install>` containing the data.
+The mapping is :ref:`configurable<ap-verify-configuration-dataset>`.
+
+.. _ap-verify-run-output:
+
+How to Run ap_verify in a New Workspace
+---------------------------------------
+
+Using the :ref:`HiTS 2015 <ap_verify_hits2015-package>` dataset as an example, one can run ``ap_verify`` as follows:
+
+.. TODO: why dataIdString instead of id or dataId? (DM-12853)
+
+.. code-block:: sh
+
+   $ python ap_verify/bin/ap_verify.py --dataset HiTS2015 --output workspace/hits/ --dataIdString "visit=54123 ccd=25 filter=g" --silent
+
+Here:
+
+* ``HiTS2015`` is the :ref:`dataset name <ap-verify-dataset-name>`,
+* ``workspace/hits/`` is the location of the :ref:`Butler repository<command-line-task-data-repo-using-uris>` in which the pipeline will work,
+* ``visit=54123 ccd=25 filter=g`` is the :ref:`dataId<command-line-task-dataid-howto-about-dataid-keys>` to process, and
+* ``--silent`` disables SQuaSH metrics reporting.
+
+This will create a workspace (a Butler repository) in ``workspace/hits`` based on ``<hits-data>/data/``, ingest the HiTS data into it, then run visit 54123 through the entire AP pipeline.
+
+.. note::
```

Comment:
No need for this note, I think. You already said it's similar, and this page along with the command reference, documents what it is exactly.

Reply:
I disagree. I think trying to use compound dataIds is an easy enough mistake to make that users won't think to check the reference and read the warning there.